source/opencl_std_ext_inst.inc

```diff
@@ -53,8 +53,8 @@ ExtInst(minmag, 44, List(OperandId, OperandId))
 ExtInst(modf, 45, List(OperandId, OperandId))
 ExtInst(nan, 46, List(OperandId))
 ExtInst(nextafter, 47, List(OperandId, OperandId))
-ExtInst(pow, 48, List(OperandId, OperandId, OperandId))
-ExtInst(pown, 49, List(OperandId))
+ExtInst(pow, 48, List(OperandId, OperandId))
+ExtInst(pown, 49, List(OperandId, OperandId))
```

Comment:
@dneto0 is this file generated by `syntax_tables.patch`?  If so, then manual modification won't work.